course_content/notebooks/dask_intro.ipynb

```diff
@@ -728,13 +728,364 @@
+   "source": [
+    "Often when loading files in Iris we will want to constrain the files before loading them. Loading using constraints has already been covered in the Iris Introduction course and in the Iris documentation. We will now consider some simple examples of loading with constraints and how to use dask in this process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "\n",
+    "from dask import delayed\n",
+    "import dask.bag as db\n",
+    "import iris"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['/project/avd/iris/resources/git/iris-sample-data/sample_data/colpex.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/rotated_pole.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/A1B_north_america.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/wind_speed_lake_victoria.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/E1.2098.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/atlantic_profiles.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/ostia_monthly.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/air_temp.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/pre-industrial.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/E1_north_america.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/A1B.2098.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/space_weather.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/uk_hires.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/orca2_votemper.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/hybrid_height.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/SOI_Darwin.nc']"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "filepath = '/project/avd/iris/resources/git/iris-sample-data/sample_data'\n",
+    "files = glob.glob(os.path.join(filepath, '*.??'))\n",
+    "files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 1. Constraining by Phenomenon"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One way to constrain data is based on the phenomenon it represents. Below we have added a constraint so only cubes with the name 'air_potential_temperature' will be loaded."
```

Comment:
You could put the name in back-ticks (\`) so that the name is rendered in preformatted text instead.